multicast_expert/rx_socket.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, addr_family: int, mcast_ips: List[str], port: int, iface_ip:
             this value is processed.
         :param blocking: Legacy alias for timeout.  If set to True, timeout is set to None (block forever).  If set to
             False, timeout is set to 0 (nonblocking).
+        :param enable_external_loopback: Enable receiving multicast packets sent over external interfaces.
```

Comment:
Can we add another line to the docs like
```
If and only if this option is set to True, McastRxSockets will receive packets sent by McastTxSockets open on the same mcast address/port/interface combination.
```
Let's add this to McastTxSocket too

Reply:
Added